### PR TITLE
JD: Bugfix:Github:ID#3

### PR DESCRIPTION
**Pull request by jaydeepc**
Details:
     ** Stacktrace does not show up for passed list.
     ** For page which shows all the tests result, stacktrace link is replaced by '****'

### CODE REVIEW

**Comment by jaydeepc** [comment]
#3

File: js/test_results.js [hunk 1]
```diff
@@ -13,6 +14,12 @@ $(document).ready(function() {
         };
 
         for(i=0; i<test_list.length; i++){
+            if (test_list[i].outcome != 'passed'){
+                show_failure_trace = "</td><td class='stacktrace'><a onclick='showModal("+i+");'>Show Failure Trace</a></td><td>";
+            }
+            else{
+                show_failure_trace = "</td><td class='stacktrace'>**********</td><td>";
```
Comment (harinee) [inline js/test_results.js]: use *** NA *** instead

File: js/test_results.js [hunk 2]
```diff
@@ -61,6 +68,14 @@ $(document).ready(function() {
                         }
                     }
                 }
+        }).then(function(){
```
Comment (harinee) [inline js/test_results.js]: Not required

File: data/result.json [hunk 3]
```diff
@@ -0,0 +1 @@
...ey-solutions.com/tst2app2/price/price/api/in/forcebreakup\nE       Method: POST\nE       payload: {\"filters\": {\"dimensionfilters\": [{\"values\": [{\"key\": 41}]}], \"factIdentifiers\": [{\"key\": 9999999}]}}\nE       *****************************\nE       Response Details: \nE       *****************************\nE       Body: b'{\"success\":{\"forces\":{}},\"error\":\"\"}'\n\ntests/force_breakup_tests/test_force_breakup_api.py:92: AssertionError", "duration": 1.0963268280029297, "name": "call", "outcome": "failed"}, "duration": 1.0969104766845703, "outcome": "failed"}, {"setup": {"duration": 0.0002951622009277344, "name": "setup", "outcome": "passed"}, "teardown": {"duration": 8.511543273925781e-05, "name": "teardown", "outcome": "passed"}, "name": "tests/force_breakup_tests/test_force_breakup_api.py::ForceBreakupAPITests::test_user_can_not_retrieve_force_details_with_invalid_factid", "run_index": 12, "call": {"duration": 1.006775140762329, "name": "call", "outcome": "passed"}, "duration": 1.0074505805969238, "outcome": "passed"}], "created_at": "2017-01-30 20:31:23.175451", "summary": {"num_tests": 29, "failed": 4, "duration": 137.94698810577393, "passed": 23, "skipped": 2}}}
```
Comment (harinee) [inline data/result.json]: Remove content

**Pull request merged**